src/State/ComputedPairs.lua

```diff
@@ -114,7 +115,11 @@ function class:update(): boolean
 			table.clear(keyData.dependencySet)
 
 			local oldOutValue = oldOutput[key]
-			local processOK, newOutValue = captureDependencies(keyData.dependencySet, self._processor, key, newInValue)
+			local processOK, newOutValue = dontYield(
+				captureDependencies,
+				"computedPairsCannotYield",
```

Comment:
It's best not to pass the error message. Instead, implement a return value that could be named yielded. Then error outside of the dontYield function.